sdk/communication/azure-communication-rooms/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@
 ### Other Changes
 - The `MicrosoftBotIdentifier` and `MicrosoftBotProperties` have been deprecated.
 
+## 1.1.0 (2024-04-15)
```

Comment:
What is the 1.1.0b2 version above? Should we move those entries to this version?